src/openrct2/world/TileInspector.cpp

```diff
@@ -343,7 +343,7 @@ namespace OpenRCT2::TileInspector
                     return GameActions::Result(GameActions::Status::Unknown, STR_TOO_MANY_BANNERS_IN_GAME, STR_NONE);
                 }
                 // Copy the banners style
-                *newBanner = *GetBanner(bannerIndex);
+                newBanner = GetBanner(bannerIndex);
```

Comment:
This is no longer correct, the purpose is to actually copy the data and not change pointer